src/public/index.html

```diff
@@ -7,10 +7,34 @@
     <meta name="description" content="A blog about something awesome!">
     <title>My Blog</title>
   </head>
+  <header>
+    <h1>Something Awesome!</h1>
+  </header>
+
 
     <body>
-      <h1> Yoooooo </h1>
-      <h2> Y'all ready for this?</h2>
+      <nav>
+        <ul>
+          <a href="/About/"> About
+          </a>
+        </ul>
+        <ul>
+          <a href="/Contact/"> Contact
+          </a>
+        </ul>
+      </nav>
+
+      <h2> Y'all ready for this? </h2>
+
+      <main>
+        <h2> Latest Posts </h2>
+
```

Comment:
Missing the `<article>` elements that contain the blog post titles and excerpts.